doc/source/overview/getting_started.md

```diff
@@ -2,11 +2,112 @@
 
 ## Installation
 
-alibi-detect can be installed from [PyPI](https://pypi.org/project/alibi-detect/):
+Alibi Detect can be installed from [PyPI](https://pypi.org/project/alibi-detect/) or [conda-forge](https://conda-forge.org/)
+by following the instructions below.
+
+
+``````{dropdown} Install via PyPI
+
+```{div} sd-mb-3
+- Alibi Detect can be installed from [PyPI](https://pypi.org/project/alibi-detect/) with `pip`:
+```
+
+`````{tab-set}
+
+````{tab-item} Standard
+:sync: label-standard
+:class-label: sd-pt-0
+
+```{div} sd-mb-1
+Installation with default [TensorFlow](https://www.tensorflow.org/) backend.
+```
 
 ```bash
 pip install alibi-detect
 ```
+````
+
+````{tab-item} PyTorch
+:sync: label-torch
+:class-label: sd-pt-0
+
+```{div} sd-mb-1
+Installation with [TensorFlow](https://www.tensorflow.org/) and [PyTorch](https://pytorch.org/) backends.
+```
+
+```bash
+pip install alibi-detect[torch]
+```
+
+```{note}
+If you wish to use the GPU version of PyTorch, or are installing on Windows, it is recommended to [install and test PyTorch](https://pytorch.org/get-started/locally/) 
+prior to installing alibi-detect.
+```
+````
+
+````{tab-item} Prophet
+:class-label: sd-pt-0
+
+```{div} sd-mb-1
+Installation with the [Prophet](../od/methods/prophet.ipynb) time series outlier detector enabled.
+```
+
+```bash
+pip install alibi-detect[prophet]
+```
+````
+
+`````
+``````
+
+
+``````{dropdown} Install via conda-forge
```

Comment:
Is it possible to install via `conda` but with `prophet`? If not, then probably not worth mixing up `pip` and `conda` and just sticking with the `pip` recommended route for that functionality.

Reply:
I've not included the "prophet" tab within the "conda" drop-down for now, as haven't properly tested how well `prophet` works with conda, and also agreed that it would be confusing to be mentioning pip installs in the conda section.

![image](https://user-images.githubusercontent.com/32061685/152167641-09a0151a-8df9-42cb-a571-09e96e5da2ac.png)

We can deffo explore installing prophet with conda in the future though (or even as part of this PR if you think its important).

Reply:
Happy to leave it as is for now. I think this will come back as part of the dependency management work as we'll need to figure out how to do the options via `conda`.

Reply:
Sounds good!